__tests__/components/Footer/Footer.test.js

```diff
@@ -12,15 +12,15 @@ describe('Footer should run correctly', () => {
         const onPressUpvote = jest.fn()
         const onPressBlock = jest.fn()
         const  onPressComment = jest.fn()
-        const {toJSON, getByTestId, getAllByTestId} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />)
+        const {toJSON, getByTestId, getAllByTestId} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />, {wrapper: Store})
         expect(toJSON).toMatchSnapshot()
         expect(getAllByTestId('onPressBlock')).toHaveLength(1)
         expect(getAllByTestId('availableComment')).toHaveLength(1)
         fireEvent.press(getByTestId('onPressBlock'))
         expect(onPressBlock).toHaveBeenCalled()
-        const {getAllByTestId: getIsSelf} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={true} onPressShare={onPressShare} statusVote={'none'} />)
+        const {getAllByTestId: getIsSelf} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={true} onPressShare={onPressShare} statusVote={'none'} />, {wrapper: Store})
         expect(getIsSelf('isself')).toHaveLength(1)
-        const {getAllByTestId: getIsBlock} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: true}} loadingVote={false} disableComment={true} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />)
+        const {getAllByTestId: getIsBlock} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: true}} loadingVote={false} disableComment={true} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />, {wrapper: Store})
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are related to wrapping the Footer component with the Store context provider during rendering for testing. This is acceptable if the Footer component relies on values from the Store context. However, ensure that the Store provider is properly initialized with the necessary default values for the test environment.

```diff
- const {toJSON, getByTestId, getAllByTestId} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />)
+ const {toJSON, getByTestId, getAllByTestId} = render(<Footer totalComment={10} onPressComment={onPressComment} onPressBlock={onPressBlock} onPressDownVote={onPressDownVote} onPressUpvote={onPressUpvote} blockStatus={{blocker: null}} loadingVote={false} disableComment={false} totalVote={5} isSelf={false} onPressShare={onPressShare} statusVote={'none'} />, {wrapper: Store})
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->